packages/elements/examples/nextjs/app/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -51,23 +50,16 @@ export default function SignInPage() {
                 Sign In with Metamask
               </SignInSocialProvider>
             </div>
-
             <Hr />
-
             <div className='flex gap-6 flex-col'>
               <CustomField
                 label='Email'
                 name='identifier'
               />
 
-              {/* <Hr />
-
-              <CustomField
-                label='Phone'
-                name='identifier'
-              /> */}
-
-              <CustomSubmit>Sign In</CustomSubmit>
+              <CustomSubmit>
+                <SignInLoading fallback={<>Submitting...</>}>Sign In</SignInLoading>
```

Comment:
Example usage for rendering based on the global loading state. For further focusing we could do something like:
```
<SignInLoading context="oauth_github">
```

Ideally we can make this a non-flow-specific component, though it's hard if we rely on state node tags.